pypesto/history/base.py

```diff
@@ -70,18 +70,18 @@ def update(
 
     def finalize(
         self,
-        message: str = None,
-        exitflag: str = None,
+        message: Union[str, None] = None,
```

Comment:
For future use: this is equivalent to `Optional[str]`. Slightly shorted. Fine to leave as is.